README.md

```diff
@@ -64,11 +64,25 @@ Remember to remove the k8 cluster to avoid ONS still being charged.
 
 ### Run - Local Locust against RH in GCP
 
+Before attempting a local run you'll need to create some test data and start the shell:
+
+	$ cd census-rh-performance-test
+	$ cp test_data/example_event_data.2.txt test_data/event_data.txt 
+	$ pipenv shell
+
 If you want to run a local Locust to generate traffic for a RH which is deployed in GCP then 
 you'll need a command like:
 
     $ locust -f locust_tasks/locustfile.py --no-web --clients 750 --hatch-rate 20 --csv-full-history --csv /tmp/rhui.csv --reset-stats --host=http://34.107.206.101
 
+To do a command line run against a local RH:
```

Comment:
I tried that command in line 76 (I know this hasn't changed in the PR though) but it gave me this error: ImportError: cannot import name 'TaskSequence'

Reply:
What version of Python are you using? 
I'm on 3.7.5

Reply:
Are you doing '$ pipenv shell' before attempting to run Locust?